doc/querying.rdoc

```diff
@@ -715,6 +715,20 @@ aggregation:
 
   Album.select_group(:artist_id).select_append{sum(num_tracks).as(tracks)}
   # SELECT artist_id, sum(num_tracks) AS tracks FROM albums GROUP BY artist_id
+  
+You can combine select_group, select_append while we are using a join and a where statement:
+
+  Album.
+  left_join(:artists.
+      select_group(:id).
+      select_append{max(:sales)},
+    :id=>:album_id).
+  where{{Album[:song] => "Bohemian Rhapsody"}}  
+  # SELECT * FROM `albums` 
```

Comment:
Let's drop the MySQL/SQLite specific quoting.  Probably best to show an unquoted example.

Reply:
I've removed all back-ticks. Pls LMK if this is wrong.